osvr/RenderKit/RenderManager.h

```diff
@@ -612,8 +612,11 @@ namespace renderkit {
                 m_distortionParameters; ///< One set per eye x display
 
             bool m_enableTimeWarp;       ///< Use time warp?
+            bool m_justInTimeWarp;       ///< Use just-in-timewarp?
+                                         ///(requires enable)
+            float m_justInTimeWarpRotation; ///< Rotation in degrees, 0 = top, 90 = right
```

Comment:
Inertia from the initial implementation.  It is indeed a lot of copying of parameters from one structure to another.  My original thought was that there would be an external generator that would either construct the parameters directly or read them from the config file (perhaps an external GUI), but in practice they are always read from the config file.  You could open a new ticket to remove all of this redundancy.